indica/readers/available_quantities.py

```diff
@@ -27,15 +27,19 @@
         "zeff": "effective_charge",
     },
     "get_helike_spectroscopy": {
+        "ti_w": "ion_temperature",
+        "ti_z": "ion_temperature",
+        "te_n3w": "electron_temperature",
+        "te_kw": "electron_temperature",
         "int_w": "line_intensity",
         "int_k": "line_intensity",
         "int_tot": "line_intensity",
         "int_n3": "line_intensity",
-        "te_kw": "electron_temperature",
-        "te_n3w": "electron_temperature",
-        "ti_w": "ion_temperature",
-        "ti_z": "ion_temperature",
-        "spectra": "spectra",
+        "intens": "intensity",
```

Comment:
Do we want/need all these quantities in Indica?
Or would we rather have only the highest level quantity (i.e. emission), checking that the data is non NaN, and otherwise going further down a level until we find != NaNs? 

Also, I've never been a fan of Emission, since this means something different in the community (typically W/m^3). So I'd rather call it "spectra" in Indica, and the units will just depend on which spectra is available.